### PR TITLE
Initialize components: Milestone 1

### PR DESCRIPTION
**Pull request by pasytchangwa**
In this milestone the following was done:

- Initialized React app.
- Added React Redux (npm install react-redux).
- Structured application files using a "feature folder" approach and used the ducks pattern for Redux files.
- The building blocks of app app are set as re-usable components.
- Added React Router and set two <Route>s and <Link>s for the app's navigation:

### CODE REVIEW

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @pasytchangwa , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't meet the minimum professional README template as described by Microverse. Remember to use this [template](https://github.com/microverseinc/readme-template) and customize it according to your  project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: package.json [hunk 1]
```diff
@@ -3,13 +3,15 @@
...
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
-    "web-vitals": "^1.0.1"
+    "web-vitals": "^1.1.2"
   },
```
Comment (DeliceLydia) [inline package.json]: - [ ] Please add `react-redux` in your dependencies

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @pasytchangwa, 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

- Please add `index.js` in the redux folder

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Moving out `css` file in the component folder to make it cleaner

### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (pasytchangwa) [reply package.json]: Hello, thank you for the review. I did not understand your first request. Are you asking me to move my index.js file into the redux folder or you are asking me to create another index.js file?

Screenshot from 2021-08-31 10-55-39

The template above shows that I should have it in the src directory. 

Thank you for clarifying me

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**